components/ui/AppCheckbox.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <cv-checkbox
+    class="app-checkbox"
+    v-bind="formatedOption"
+  />
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { Component, Prop } from 'vue-property-decorator'
+
+type ckeckboxOption = {
+  label: string,
+  value: string
+}
+
+@Component
+export default class extends Vue {
+  @Prop(String) option!: string
+  // TODO: Add checked, aria-checked and @change when doing the filter behaviour. Also
+  // reuse this component on events page
+
+  formatedOption = this.formatOption(this.option)
+
+  formatOption (option: string): ckeckboxOption {
+    return { label: option, value: option }
+  }
```

Comment:
The `option` is declared as [`prop`](https://vuejs.org/v2/api/#props), i.e, an attribute that accepts data from the parent. By default, it's not reactive. If you want to modify a value on the parent and see how the child catches the modification, use [@ProvideReactive](https://github.com/kaorun343/vue-property-decorator#-providereactivekey-string--symbol--injectreactiveoptions--from-injectkey-default-any---injectkey-decorator).

On the other hand, as you can read [here](https://class-component.vuejs.org/guide/class-component.html#data), we are creating the `formattedOption` variable as component data. And that means that it will be reactive unless its value is undefined.

So, in this code, `option` is not reactive, but `formattedOption` is.

Does this answer your question?